superdesk/io/feeding_services/ftp.py

```diff
@@ -79,8 +97,14 @@ def _update(self, provider, update):
             except KeyError:
                 logger.warning('missing move_path')
                 move_dest_path = None
+            try:
+                move_dest_path_error = os.path.join(config.get('path', ''), config['move_path_error'])
```

Comment:
I would prefer the default and create that folder if it's not there, but lets see what karel thinks, or @hlmnrmr ?